packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -94,7 +92,13 @@ export class ClientMemoryState {
     this.queryMapping = new ObjectMap<Target, TargetId>(t => t.canonicalId());
     this.limboMapping = {};
     this.activeTargets = {};
-    this.limboIdGenerator = TargetIdGenerator.forSyncEngine();
+    this.nextLimboTarget = 1;
+  }
+
+  nextLimboTargetId(): TargetId {
```

Comment:
nit: Naming the `nextLimboTargetId()` method the same as the variable name somewhat implies that it is a simple "getter" method.  Consider renaming to something that implies that it is "doing something" not just "getting something".  e.g. `generateLimboTargetId()`.